lib/src/main/java/fi/epicbot/toster/executor/ShellExecutor.kt

```diff
@@ -15,6 +15,10 @@ class ShellExecutor(projectDir: String, private val logger: ShellLogger) {
         makeDir(workingDir.toString(), clearBefore = true)
     }
 
+    suspend fun delay(delayMillis: Long) {
+        kotlinx.coroutines.delay(delayMillis)
```

Comment:
Why not move to inport?

Reply:
with import it will recursion.
leave as is